src/server/apis/user.ts

```diff
@@ -65,9 +68,65 @@ export async function handleRemoveRepo(
   return responseHelper.data({status: 'ok'});
 }
 
+export async function handleAddRepo(
+    request: express.Request,
+    userRecord: UserRecord): Promise<APIResponse<api.GenericStatusResponse>> {
+  const text = request.body.text;
+
+  if (!text) {
+    return responseHelper.error('invalid-request', 'No text found');
```

Comment:
Use unique codes for errors - just makes future debugging easier.

Reply:
Done.